pom.xml

```diff
@@ -106,6 +106,16 @@
     </plugins>
     <pluginManagement>
       <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+          <!--
+          maven totally change his way to copy symlink
+          this mean the symlink is copied as a symlink and not anymore as a new file
+          see JENKINS-59291
+          -->
+          <version>2.7</version>
```

Comment:
So, this is to keep us from using `3.1.0` or is it meant to make us use at least `2.7`?

Reply:
Ah: 
https://stackoverflow.com/questions/40346225/maven-resources-plugin-symbolic-link-handling
https://issues.apache.org/jira/browse/MRESOURCES-237

Reply:
Is there anyway to test this change within this plugin?  Can we add a test the checks the symlink behavior so it doesn't break in the future?